builder/amazon/chroot/run_local_commands.go

```diff
@@ -0,0 +1,37 @@
+package chroot
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/post-processor/shell-local"
```

Comment:
Yes, I agree this should be deduplicated and moved into `communicator/shell-local`. In the meantime I think this import is fine. It makes it simpler to update when we take care of the communicator than doing another copy.